contracts/modules/royalty/policies/RoyaltyPolicyLAP.sol

```diff
@@ -125,21 +127,12 @@ contract RoyaltyPolicyLAP is IRoyaltyPolicyLAP, Governable, ERC1155Holder, Reent
             // called _initPolicy() when linking to their parents with onLinkToParents() call.
             address[] memory rootParents = new address[](0);
             bytes[] memory rootParentRoyalties = new bytes[](0);
-            _initPolicy(ipId, rootParents, rootParentRoyalties, externalData);
+            _initPolicy(ipId, rootParents, rootParentRoyalties);
         } else {
-            InitParams memory params = abi.decode(externalData, (InitParams));
             // If the policy is already initialized and an ipId has the maximum number of ancestors
             // it can not have any derivative and therefore is not allowed to mint any license
-            if (params.targetAncestors.length >= MAX_ANCESTORS)
+            if (royaltyData[ipId].ancestorsAddresses.length >= MAX_ANCESTORS)
```

Comment:
This can be `data.ancestorsAddresses` since we declare the same above as a variable

Reply:
https://github.com/storyprotocol/protocol-core-v1/pull/4/commits/31917e6da5cb27816ae0161c263b68c18cdf10d1